bakery/src/tests/test_cli.js

```diff
@@ -69,13 +90,13 @@ test('build pipelines', async t => {
         ],
         {
           // Include credentials in environment for local pipelines
```

Comment:
Minor: I think this comment can be removed

Reply:
I'm not sure I understand. The reason those creds need to be there is still for the local pipeline to not fail, right?

Reply:
Oh, perhaps I didn't read this right when looking at it earlier. Is this now passing those for all environments (e.g. the change to remove `env === 'local'`)?

Reply:
correct, it should be fine to include the environment variables for any pipeline, although, I'm just now realizing I should add a test to check that non-local pipelines do not use the env vars if present (since they would be inlined and readable with fly)

Reply:
Okay, added that test.